contracts/Auction.sol

```diff
@@ -46,158 +45,179 @@ contract Auction {
         reserve_price = starting_price;
         ended = false;
-        current_highest_bid = 0; 
+        current_highest_bid = 0;
     }
 
-
-
-    function viewItem() public {
+    function viewItem() public view returns (string memory, string memory) {
         // Leads to a page of the item which includes information like name, description, time remaining, bidding history.
         // Has helper functions.
-    }
+        string memory name = this.getName();
+        // string memory conjunction = ": "
+        string memory desc = this.getDescription();
 
+        return (name, desc);
+    }
 
-    function getName() public returns(string memory) {
+    function getName() public view returns (string memory) {
         // Function: Returns the name of the item.
         return item_name;
     }
 
-
-    function getDescription() public returns(string memory) {
+    function getDescription() public view returns (string memory) {
         // Function: Returns description of the item.
         return description;
     }
 
-
-    function getTimeRemaining() public returns(uint) {
+    function getTimeRemaining() public view returns (uint256) {
         // Condition(s): Auction cannot be over. Auction has not ended.
         // Function: Returns time remaining in the auction.
 
-        require(block.timestamp <= end_time,"Auction has already ended.");
+        require(block.timestamp <= end_time, "Auction has already ended.");
         require(!ended, "Auction has already ended.");
 
-        uint time_remaining = end_time - block.timestamp;
+        uint256 time_remaining = end_time - block.timestamp;
         return time_remaining;
     }
 
-
-    function getBidHistroyCount() public returns (uint) {
+    function getBidHistroyCount() public view returns (uint256) {
         return bid_history.length;
     }
 
-
-    function getBidHistoryByIndex(uint index) public returns (address, uint, uint) {
-        bid_record storage bid = bid_history[index];
-        return (bid.bidder_address, bid.bid_amount, bid.time_placed);
+    function getBidHistoryByIndex(uint256 index)
+        public
+        view
+        returns (
+            address,
+            uint256,
+            uint256
+        )
+    {
+        // ;
+        return (
+            bid_history[index].bidder_address,
+            bid_history[index].bid_amount,
+            bid_history[index].time_placed
+        );
     }
 
     /* 
         transfer funds (withdraw the amount) from the buyer address
         transfer it to the seller address 
     */
-    function transferFunds(address buyer, address seller, fixed128x10 cost) public {
-        
-        buyer.transfer(cost);
-        buyer(this).balance -= cost;
-        
-        seller(this).balance += cost
-        
+
+    function transferFunds(
+        // address payable buyer,
+        address payable seller,
+        uint256 cost
+    ) public {
+        seller.transfer(cost);
+        // buyer(this).balance -= cost;
```

Comment:
if you already fixed the issue, no need to leave the deprecated here, you could just remove them instead of commneting

Reply:
this issue is not fixed. Author has not clarified use of syntax. and there are not other function to reduce and increase respective balance from the addresses